### PR TITLE
Cleared some unintelligibles from changelog-330

### PR DESCRIPTION
**Pull request by MissMeg**
Cleared some unintelligibles from changelog-330

Let me know if there's anything I need to change 👍

### CODE REVIEW

File: podcast/the-changelog-330.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ For data analysts - if you tell a data analyst "Oh yeah, you should use Git in o
...
 
-The idea of being able to tell them, "Well, actually all of this source code - let's see which parts are gonna be the easiest ones, \[unintelligible 00:21:58.18\] modernizing traditional applications", which is not really cloud-native, but we can make it be cloud-native, we can make it run in Kubernetes... And then what is the COBOL that -- you know, that's gonna be an interesting challenge to migrate. So having a view of all of this by just running a couple queries is really powerful. The other option is literally running -- I'm gonna be very hopeful and say that you could run a really huge Bash\[unintelligible 00:22:25.25\] calling Git very often, and maybe you will get something similar... But it would take hours instead of seconds.
```
Comment (jerodsanto) [inline podcast/the-changelog-330.md]: Highly likely that the word phrase used here was `huge Bash script` not `huge Bash crib` 😁

File: podcast/the-changelog-330.md [hunk 2]
```diff
@@ -78,15 +78,15 @@ That is super-useful, because basically the whole idea is that it brings visibil
...
 
 **Adam Stacoviak:** You can look up the repo URL on GitHub, even if it's a private GitHub repo as well.
 
 **Francesc Campoy:** Yeah, yeah. So the cool thing is that--
 
 **Adam Stacoviak:** Because you have teams at the org level, so you could look up not so much by the repo, but by the repo URL.
 
-**Francesc Campoy:** Yeah, yeah. So the thing is that all of that -- that is the GitHub API. We work with any Git repository, so all of the concepts that we work with are Git, for now... That's why, for instance, the organization is a GitHub or GitLab \[unintelligible 00:15:39.12\] you could expose it from a different dataset. Just download the whole thing, put it in a MySQL and that's it. You can do that, too.
+**Francesc Campoy:** Yeah, yeah. So the thing is that all of that -- that is the GitHub API. We work with any Git repository, so all of the concepts that we work with are Git, for now... That's why, for instance, the organization is a GitHub or GitLab. Point you could expose it from a different dataset. Just download the whole thing, put it in a MySQL and that's it. You can do that, too.
```
Comment (jerodsanto) [inline podcast/the-changelog-330.md]: Maybe give this one another listen? `Point you could expose` doesn't make much sense when I read it? Could be what he said, but...

Comment (pnevares) [reply podcast/the-changelog-330.md]: I took a look at this, does sound like it's what he's saying. I've formatted a bit better
> "Point: you could expose it..."

File: podcast/the-changelog-330.md (hunk 1 above)
Comment (pnevares) [reply podcast/the-changelog-330.md]: Updated :)

**Pull request merged**